include/RAJA/pattern/params/forall.hpp

```diff
@@ -285,13 +285,16 @@ template<std::size_t name_idx,
          std::enable_if_t<(name_idx >= sizeof...(Args))>* = nullptr>
 std::string get_kernel_name_string(camp::tuple<Args...>&& tuple_args)
```

Comment:
@LLNL/raja-core , if I try to add the RAJA_UNUSED_VAR within the function signature I get compilation errors -- function stops being well defined. Does anyone know why?

Reply:
I think you want to use this instead: https://github.com/LLNL/RAJA/blob/develop/include/RAJA/util/macros.hpp#L95